src/main/java/net/mcreator/element/parts/gui/Button.java

```diff
@@ -29,16 +29,18 @@ public class Button extends SizedComponent {
 
 	public String name;
 	public String text;
+	public boolean isTextButton;
 	public Procedure onClick;
 	public Procedure displayCondition;
 
 	public Button(String name, int x, int y, String text, int width, int height, Procedure onClick,
-			Procedure displayCondition) {
+			Procedure displayCondition, boolean isTextButton) {

```

Comment:
Please refactor the code part too, not just the string in UI. At this point we still have chance to call it properly in all places

Reply:
I kept it this way because Mojang never uses the word "undecorated", so I thought it would be easier to understand what's the purpose of the parameter/where/how it's used in templates.